javascript/javascript1/week3/ex-4.js

```diff
@@ -0,0 +1,40 @@
+//  Exercise 4: NOnoN0nOYes (Note taking app)
+
+const notes = [];
+
+function saveNote(content, id) {
+	const note = { content, id };
+	notes.push(note);
+}
+
+function getNote(id) {
+	for (let i = 0; i < notes.length; i++) {
```

Comment:
Nice and readable implementation! ⭐ The assignment also wants you to come up with a new feature which seems to be missing here. 🔍 

You have used `map` extensively until now; can you find a similar method that could solve this without a for loop? 😄